samples/openapi3/client/petstore/python-aiohttp/tests/test_rest.py

```diff
@@ -0,0 +1,73 @@
+import os
+import unittest
+from unittest.mock import AsyncMock, Mock, patch
+
+
+import petstore_api
+
+
+def get_field_from_formdata(formdata, name):
+    return next(filter(lambda x: x[0]["name"] == name, formdata._fields))[-1]
+
+
+class TestMultipleResponseTypes(unittest.IsolatedAsyncioTestCase):
+    def setUpFiles(self):
+        self.test_file_dir = os.path.join(os.path.dirname(__file__), "..", "testfiles")
+        self.test_file_dir = os.path.realpath(self.test_file_dir)
+        self.test_file_path = os.path.join(self.test_file_dir, "foo.png")
+
+    def setUp(self):
+        self.api_client = petstore_api.ApiClient()
```

Comment:
This must be called in an asynchronous context with:

```python
async with petstore_api.ApiClient() as api_client:
```

or within an async setup function (and you need to explicitly close the client afterwards):

```python
    async def asyncSetUp(self):                                                                                                                                                                                     
        self.api_client = petstore_api.ApiClient()                                                                                                                             
        self.fake_api = petstore_api.FakeApi(self.api_client)                                                                                                                         
        self.setUpFiles()                                                                                                                                                     
                                                                                                                                                                                                                           
    async def asyncTearDown(self):                                                                                                                                                                     
        await self.api_client.close()    
```

Reply:
Good catch, thanks. Updated in ededfd5